salt/modules/parted.py

```diff
@@ -671,13 +674,14 @@ def toggle(device, partition, flag):
     '''
     partition.toggle device partition flag
 
-    Toggle the state of <flag> on <partition>
+    Toggle the state of <flag> on <partition>. Valid flags are the same as 
```

Comment:
Could you please remove the trailing whitespace from this line in order to satisfy the lint tests? Thanks.